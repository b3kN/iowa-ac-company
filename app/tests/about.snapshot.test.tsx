/**
 * @jest-environment jsdom
 */

import About from "@/about/page";
import { ChakraProvider } from "@chakra-ui/react";
import { render } from "@testing-library/react";

it("renders homepage unchanged", () => {
  const { container } = render(
    <ChakraProvider>
      <About />
    </ChakraProvider>,
  );

  expect(container).toMatchSnapshot();
});
