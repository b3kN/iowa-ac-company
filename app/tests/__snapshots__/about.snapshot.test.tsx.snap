// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="css-19cop37"
  >
    <h2
      class="chakra-heading css-1sq2t96"
    >
      About Bob's
    </h2>
    <p
      class="chakra-text css-0"
    >
      Established in 2003, 
       offers quality and professional services to Cedar Rapids, IA, and the surrounding area. Our services include the repair and installation of air conditioners and furnaces. We understand the importance of heating and cooling for your home, and our friendly staff will gladly attend to your needs. Give us a call today.
    </p>
  </div>
  <div
    class="css-1unuuic"
  >
    <div
      class="chakra-stack css-1rdhlq"
    >
      <div
        class="chakra-stack css-1yuug1o"
      >
        <p
          class="chakra-text css-1xrzlq"
        >
          Lorem ipsum dolor sit amet consectetur adipiscing elit sed do eiusmod tempor
        </p>
        <div
          class="chakra-stack css-1f0wxn3"
        >
          <button
            class="chakra-button css-1hn8sdm"
            type="button"
          >
            Schedule Service
          </button>
          <button
            class="chakra-button css-1dm6egg"
            type="button"
          >
            Contact Us
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    class="css-19cop37"
  >
    <h3
      class="chakra-heading css-ycmg1h"
    >
      Success Stories
    </h3>
    <p
      class="chakra-text css-zlipk4"
    >
      Mauris id lacus vitae mauris tempor fermentum id sed leo. Donec hendrerit ex mauris, lobortis laoreet tellus gravida a. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Aenean dignissim accumsan tempor. Vestibulum condimentum mattis diam et aliquam. Aliquam erat volutpat. Vestibulum blandit gravida sagittis.
    </p>
    <p
      class="chakra-text css-0"
    >
      Curabitur ac tellus sodales, bibendum neque et, ullamcorper nibh. Sed eleifend cursus lacus. Pellentesque ultricies condimentum sodales. Aenean vel gravida odio. Aliquam ultricies tellus eget nunc imperdiet, ac rutrum sapien finibus. Vestibulum ac massa dolor. Sed quam mauris, vulputate vitae rutrum sed, efficitur ac orci.
    </p>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
